services/QuillLMS/.env-sample

```diff
@@ -21,7 +21,7 @@ SEGMENT_WRITE_KEY=
 MIXPANEL_KEY=
 REDISCLOUD_URL=redis://localhost:6379/0
 QUILL_API_URL=https://localhost:3000/api/v1
-DEFAULT_URL=http://localhost:3000
```

Comment:
3000 is the default for Rails so why the switch to 5000?

Reply:
QUILL_API_URL is not used, so I'm deleting that regardless, thanks for mentioning. 

3000 is the default port for Rails, but not the default port for local LMS. To expound:

- The correct/consensus way to boot local LMS is via Foreman ([link](https://quill.slack.com/archives/C03UVPPNH/p1704384140237229))

- Foreman defaults to process #1 in a Procfile to run on port 5000, auto-incrementing port numbers for additional processes. ([link](https://github.com/ddollar/foreman/blob/3a262714030a83a59388a48ffdb2e93cdac12388/lib/foreman/engine.rb#L24))

Thus, I believe the most logical port to run the CircleCI Rails server on is 5000. This way, if you're debugging an e2e test on CircleCI while running Rails locally, you don't have to mentally move between ports.

Reply:
Thank you for the context.  I was wondering where that 5000 came from since I've noticed it pop-up seemingly randomly when I set puma to 3000.